rdt-open-reader/src/test/java/io/ona/rdt/presenter/PatientRegisterFragmentPresenterTest.java

```diff
@@ -116,7 +116,7 @@ public void testGetMainConditionShouldReturnCorrectMainCondition() {
     @Test
     public void testMainColumnsShouldReturnCorrectMainColumns() throws Exception {
         assertEquals(new String[]{RDT_PATIENTS + "." + "relationalid", RDT_PATIENTS + "." + FIRST_NAME,
-                RDT_PATIENTS + "." + LAST_NAME, RDT_PATIENTS + "." + AGE, RDT_PATIENTS + "." + SEX, RDT_PATIENTS + "." +  PATIENT_ID},
+                RDT_PATIENTS + "." + LAST_NAME, RDT_PATIENTS + "." + AGE, RDT_PATIENTS + "." + SEX, RDT_PATIENTS + "." +  PATIENT_ID, RDT_PATIENTS + "." + Constants.DBConstants.DOB},
```

Comment:
I like the dot, might as well move our dot to an importable file and reuse that here, https://github.com/onaio/rdt-standard/pull/456/files#diff-47171e72d37a66a09b38706fa0eaf22bR25

also can we cleanup here to use a string join

Reply:
fixed.